### PR TITLE
Clean up organizers code

### PR DESCRIPTION
**Pull request by matthew-gerstman**

### CODE REVIEW

File: src/pages/home/Organizers.tsx [hunk 1]
```diff
@@ -1,64 +1,68 @@
...
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            <div>
-              <img src={matthew2} /> <p>Matthew Gerstman</p>
-            </div>
-          </a>
-        </li>
+        {Organizers.map((organizer) => (
+          <Organizer {...organizer} key={organizer.name} />
```
Comment (donavon) [inline src/pages/home/Organizers.tsx]: in theory, 2 organizers _could_ have the same name. a better (and guaranteed unique) `key` would be their twitter handle. Just sayin' 🤷‍♀️

File: src/pages/home/Organizers.tsx [hunk 2]
```diff
@@ -1,64 +1,68 @@
...
+import matthew from "../../assets/matthew@2x.png";
 import Section from "../../components/Section";
 import TextGroup from "../../components/TextGroup";
 
+type Organizer = {
+  twitter: string;
+  image: string;
+  name: string;
+};
+const Organizers: Organizer[] = shuffle([
```
Comment (donavon) [inline src/pages/home/Organizers.tsx]: forgive the stupid question 'cuz I don't know Gatsby, but won't this shuffle on each _build_ not on each page refresh?

Comment (matthew-gerstman) [reply src/pages/home/Organizers.tsx]: It ends up shuffling on each page load when react mounts

Comment (donavon) [reply src/pages/home/Organizers.tsx]: i thought gatsby was a *static* site generator. if so, it _should_ produce static html server side at build time. therefore, there is no react and component mount on the client to shuffle.

if not, how is this any different than a standard react app?

**Review by donavon — APPROVED** [review]

**Pull request merged**